data/language/en-GB.txt

```diff
@@ -4193,7 +4193,7 @@ STR_5881    :Non-selected only
 STR_5882    :Custom currency
 STR_5883    :Custom currency configuration
 STR_5884    :{WINDOW_COLOUR_2}Exchange rate: 
-STR_5885    :{WINDOW_COLOUR_2}is equivalent to {COMMA32} GBP (£)
+STR_5885    :{WINDOW_COLOUR_2}is equivalent to {COMMA32} GBP (�)
```

Comment:
Not even sure tbh... it appears in the edit mode just fine and when i try to copy and paste the older symbol in githubs online editor, it doesn't read it as a change so it won't allow fixing it. I'm going to try fixing it in VS since theres now merge conflicts.